### ISSUE TITLE
Config profiles are exactly the same

### ISSUE THREAD

**Issue by chelming** [pbowden-msft/MAUCacheAdmin#16]
I diffed the two config profiles but there's no differences between them. Is that supposed to be the case?

**Comment by chelming** [comment]
See also:
```
MD5 (com.microsoft.autoupdate2-prod.plist) = 6d2d8bdd3d07f6216dff9b6e54934fa7
MD5 (com.microsoft.autoupdate2-test.plist) = 6d2d8bdd3d07f6216dff9b6e54934fa7
```

**Comment by kevinmcox** [comment]
They're not supposed to be, I can PR a fix if Paul doesn't want to just snag it.

The Test example should contain:

`<key>ManifestServer</key>`
`<string>http://manifest-server-url/MAU/cache/collateral/Testing/</string>`

**Comment by kevinmcox** [comment]
Corrected in #17.
